website/docs/api/lookups.md

```diff
@@ -215,12 +215,11 @@ the file doesn't exist.
 
 ## Table {#table tag="class, ordererddict"}
 
-A table in the lookups. Subclass of `OrderedDict` that implements a slightly
-more consistent and unified API and includes a Bloom filter to speed up missed
-lookups. Supports **all other methods and attributes** of `OrderedDict` /
-`dict`, and the customized methods listed here. Methods that get or set keys
-accept both integers and strings (which will be hashed before being added to the
-table).
+A table in the lookups. Subclass of `dict` that implements a slightly more
+consistent and unified API and includes a Bloom filter to speed up missed
+lookups. Supports **all other methods and attributes** of `dict`, and the
+customized methods listed here. Methods that get or set keys accept both
+integers and strings (which will be hashed before being added to the table).
 
```

Comment:
Table is still implementing OrderedDict, but I guess maybe that's an implementation detail that you want to keep out of the docs ?

Reply:
Ahh, good catch – I had initially replaced `OrderedDict` with `dict`, but it didn't work for some really confusing reason (or maybe there's something I missed, not sure). So I ended up changing it back to `OrderedDict` for now.